security/key-management.md

```diff
@@ -0,0 +1,50 @@
+Management of Non-Secret Keys and Sensible Configuration Values
+===============================================================
+
+## Purpose
+
+The purpose of this document is to outline the different types of Keys and Configuration that might not be considered 
+secrets. They do not provide access to sensitive data or critical functionalities and is considered low risk if exposed. 
+While it is not classified as a secret, it is still preferred to manage it properly to maintain best practices in code 
+management and security hygiene.
+
+## Guidelines for Management
+
+### Storage
+- **Environment Variables**: Store non-secret API keys in environment variables rather than hardcoding them in the 
+  source code. This helps in maintaining cleaner code and reduces the risk of accidental exposure.
+- **Configuration Files**: If using configuration files, ensure they are not committed to version control. 
+ Use .gitignore file to exclude these configuration files from the repository.
+- **Secret Management Tools**: By using secret management tools, you can ensure that this values are securely stored
+  outside the codebase.
+
```

Comment:
Maybe worth also adding that it can make the code more easily transferrable between environments ie Dev/Test/Prod ?